src/widgets/calendar/lv_calendar.c

```diff
@@ -83,6 +83,18 @@ void lv_calendar_set_day_names(lv_obj_t * obj, const char * day_names[])
     lv_obj_invalidate(obj);
 }
 
+void lv_calendar_set_year_list(lv_obj_t * obj, const char * years_list)
```

Comment:
It should be added to `lv_calendar_header_dropdown.c` as `lv_calendar_header_dropdown_set_year_list()` (Not the best name, ideas are welcome :slightly_smiling_face: )

Reply:
Done in d7239cd, not the most beautiful name but it reflects what it does :smile: